### ISSUE TITLE
Fog of War removal Enhancement

### ISSUE THREAD

**Issue by Road-block** [AzeriteTeam/ClassicWorldMapEnhanced#1]
If this had an option to remove Fog of War (undiscovered map areas) I'd gladly ditch Mapster for it 😈

**Comment by goldpaw** [comment]
I don't wish to add that, as I think discovery is a vital part of the game.

**Comment by Road-block** [comment]
Fair enough, I can respect that. (but I'm still going to give my 2c below* 😛 )

\* _But then so is zone level, one could say it adds an element of danger if you walk into a zone and see all ☠️ levels._

_Or as it happens when you are traversing the same zones your main has discovered with an alt._

**Comment by goldpaw** [comment]
Valid arguments. 

I'd still say that zone levels help you decide where to explore, while anti-fogging removes the need to explore. But that's not really fully true either. You still have to go there if there's something that needs to be done there. And it's not like I didn't explore all of this back in 2005 anyway. 

Maybe a system where the unexplored areas sort of pulsate? Slightly fade out and in? A way to indicate they're not explored, but still allow us to see them? Because (forcefully) showing what is unexplored is kind of important to me. And I make addons for me! :smile: 

Let's re-open this one! :)

**Comment by Road-block** [comment]
Addons that did this kind of thing in the past basically tinted the unexplored areas a different hue (or just desaturate) so you know you haven't been there but you can see landmarks / subzone labels.

**Comment by goldpaw** [comment]
Sounds like a plan. :+1:

**Comment by Andoido1** [comment]
Fuking hate MAPSTER.. its so buggy... i cant click zones.

Somebody needs to developor relase a map like... Cartographer. lol Fog of War needs to go bye bye
